### PR TITLE
Add seatmap multiple selection

### LINKED ISSUE #1819: Allow seatmap multiple seat selection with one click
Add the capability to the seatmap sample to select more than a seat at once.

### PR DESCRIPTION
**Pull request by kul3r4**
Fixes #1819 
@sebastianbenz PTAL. As this is similar to the other seatmap sample, I have removed most of the docs and added the relevant one.
If you want to see it in action, you can use https://amp-demos.glitch.me/seatmap-multiple-selecting.html#development=1 and select the first or second seat on the top left row: on glitch I haven't added all the adiacent seats for all the seat, but this is done in seatmap-neighbours.json in the sample here.

### CODE REVIEW

File: src/30_Advanced/SeatMap_Multiple_Selection.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    <amp-state id="seatMultiSelection" src="/json/seatmap-neighbours.json">
+      <script type="application/json">
+      {
+      "selectedSeats": [],
+      "numberOfSeats": 3
+      }
+      </script>
+    </amp-state>
+
+    <!-- ## Seatmap with multiple selection implementation -->
```
Comment (sebastianbenz) [inline src/30_Advanced/SeatMap_Multiple_Selection.html]: suggestion: remove `implementation` from the title

Comment (kul3r4) [reply src/30_Advanced/SeatMap_Multiple_Selection.html]: done

File: src/30_Advanced/SeatMap_Multiple_Selection.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    <!-- 
+      Everytime a seat is selected, we automatically select the 2 seats on the right of it. 
+      In case the number of seats on the right of the selected one are less than 2, we only select the remaining on the line.
+    -->
+    <div>
+     <h1>Seat map</h1>
```
[the remark this answers is not in the document]
Comment (kul3r4) [reply src/30_Advanced/SeatMap_Multiple_Selection.html]: done

File: src/30_Advanced/SeatMap_Multiple_Selection.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  <body>
+    <!-- ## Seatmap state -->
+    <!-- Here we are using an `amp-state` called `seatMultiSelection` to keep all the useful information for the seat selection.
+      * we load a list of neighbours seats from the server
+      * `selectedSeats` contains a list of selected seat ids
+      * `numberOfSeats` contains the number of seats we want automatically select -->
+    <amp-state id="seatMultiSelection" src="/json/seatmap-neighbours.json">
+      <script type="application/json">
+      {
+      "selectedSeats": [],
```
[the remark this answers is not in the document]
Comment (kul3r4) [reply src/30_Advanced/SeatMap_Multiple_Selection.html]: done

**Pull request merged**